README.md

```diff
@@ -201,19 +201,19 @@ Now that you have confirmed that you do have the real SeedSigner Project's Publi
 ```
 shasum -a 256 --ignore-missing --check seedsigner.0.6.*.sha256  
 ```
-
-**On Windows (inside Powershell):** Run this command
+Wait up to 30 seconds for the command to complete, and it should display:
```

Comment:
this line can now be removed as the 0.6.x hash calculation is instant. See my comment below on moving downwards, however .